### PR TITLE
Lighten row of a new package

### PR DESCRIPTION
**Pull request by annapakula**
- Lighten row for 3 seconds of a new package
- Add new row from input with clicked 'Enter'

### CODE REVIEW

**Comment by adam-sas-on** [comment]
Quite a lot of style changes ;-)

**Comment by annapakula** [comment]
Autoformatter :P

**Review by adam-sas-on — APPROVED** [review]
Auto-formatter loves it's job ;) :D

File: onboarding/src/Components/hooks/Packages.js [hunk 1]
```diff
@@ -5,78 +5,132 @@ import PackagesRow from "../PackagesList/PackagesRow";
...
-			}
-		);
-	}, [props.count]);
+  let url = getPath(),
+    fetchProps = {
+      method: "GET",
+      headers: {
+        Accept: "application/json",
+        "Content-Type": "application/json",
+        "X-CSRFToken": "",
```
Comment (maciejjankowski) [inline onboarding/src/Components/hooks/Packages.js:20]: ma być puste?

**Review by maciejjankowski — APPROVED** [review]

File: onboarding/src/Components/hooks/Packages.js (hunk 1 above)
Comment (adam-sas-on) [reply onboarding/src/Components/hooks/Packages.js:20]: It seems that when you have a GET method
then you don't need token (I'm not sure why)

**Pull request merged**